tests/DCPS/Observer/TestObserver.h

```diff
@@ -60,7 +60,8 @@ class TestObserver : public virtual OpenDDS::DCPS::Observer
 
   template<typename Qos>
   static std::string qos_str(const Qos& qos) {
-    std::ostringstream o; o << " qos.user_data: ";
+    std::ostringstream o;
+    o << " qos.user_data: ";
```

Comment:
It would be preferrable to have unrelated changes in a separate PR from one that adds a new feature.